feat: Kubernetes storage integration tests doc (#98)
dexidp/website: content/docs/development/integration-tests.md

  ```
- clean up the etcd container: `docker rm -f dex-etcd`

## Kubernetes

> Running integration tests for Kubernetes storage require the `DEX_KUBERNETES_CONFIG_PATH` environment variable
> with the path to kubeconfig file of the existing cluster. For tests, it is ok to use "mini" Kubernetes distributive, e.g., [KinD][kind], [Microk8s][microk8s].

Reviewer: I'm not sure I understand this first sentence exactly. I think this fixes it, but am not sure if it is exactly true as per the actual config requirements:

```suggestion
Running integration tests for Kubernetes storage requires the `DEX_KUBERNETES_CONFIG_PATH` environment variable
be set with the path to kubeconfig file of the existing cluster. For tests, it is ok to use "mini" Kubernetes distributive, e.g., [KinD][kind], [Microk8s][microk8s].
```

> There is an example for running tests in the KinD cluster.

Reviewer: ```suggestion
Example KinD cluster test run:
```

> * Install KinD using the instruction from the [official website][kind-install].

Reviewer: ```suggestion
* Install KinD using the instructions from the [official website][kind-install].
```
